apps/ledger-live-desktop/tests/page/account.page.ts

```diff
@@ -121,4 +129,12 @@ export class AccountPage extends AppPage {
     expect(advancedLogsJson).toHaveProperty("index", indexNumber);
     await this.closeModal.click();
   }
+
+  @step("Expect token to be present")
+  async expectTokenToBePresent(token: Token) {
+    await expect(this.tokenList).toBeVisible();
```

Comment:
Why aren't we getting the specifiq row of the token ?

Reply:
I'm just checking that the token is in the list and then verifying that the token name and the tokenTicker is in this list

Reply:
I'm first checking that there is a list of token in the Account.
Then I check that this list contain all the data of the token that we just add (name & ticker)